doc/self-hosted/cros-install.md

```diff
@@ -0,0 +1,57 @@
+# Installng code-server in your ChromiumOS/ChromeOS/CloudReady machine
+
+This guide will show you how to install code-server into your CrOS machine.
+
+Before everything else, you are required to turn on developer mode for your machine, but for ChromiumOS and CloudReady users (Home Edition), this is done for you.
+
+## Using Crostini
+
+One of the easier ways to run code-server is via Crostini, the Linux apps support feature in CrOS. Make sure you have enough RAM, HDD space and your CPU has VT-x/ AMD-V support. If your chromebook has this, then you are qualified to use Crostini.
+
+If you are running R69, you might want to enable this on [Chrome Flags](chrome://flags/#enable-experimental-crostini-ui). If you run R72, however, this is already enabled for you.
+
+After checking your prerequisites, follow the steps in the [the install guide](index.md) on installing code-server. Once done, make sure code-server works by running it.
+
+After confirming it runs, head over to `crosh`, the native terminal for your Chromebook. Reiterating, you must have developer mode enabled to be able to enter `crosh`.
+
+Once in `crosh`, type `vsh termina`. It should drop you to the Termina's shell.
+
+```bash
+Loading extra module: /usr/share/crosh/dev.d/50-crosh.sh
+Welcome to crosh, the Chrome OS developer shell.
+
+If you got here by mistake, don't panic!  Just close this tab and carry on.
+
+Type 'help' for a list of commands.
+
+If you want to customize the look/behavior, you can use the options page.
+Load it by using the Ctrl+Shift+P keyboard shortcut.
+
+crosh> vsh termina
```

Comment:
same `termina` comment

Reply:
(This is showing as resolved, but the typo is still there (the other ones are ok, tough).